### PR TITLE
add custom typescript declaration for merge-stream

### PR DESCRIPTION
**Pull request by FredKSchott**
After merging I was still getting errors from `merge-stream` for not having a type declaration, so I bit the bullet and added one. After this polymer-cli should be building without errors.

/cc @rictic @justinfagnani

### CODE REVIEW

File: custom_typings/merge-stream.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+declare module "merge-stream" {
+
+  module mergeStream {
+    interface MergedStream extends NodeJS.ReadWriteStream {
+      add: (source: NodeJS.ReadableStream) => MergedStream;
+    }
+	}
```
Comment (rictic) [inline custom_typings/merge-stream.d.ts]: whitespace. this closing curly brace is over indented

related: clang-format? automatically eliminate almost all alignment angst!

see polymer-analyzer for how to set it up, it's very easy and very nice

Comment (FredKSchott) [inline custom_typings/merge-stream.d.ts]: oof, gross. Fixed here, will look into clang-format!

File: src/build/build.ts [hunk 2]
```diff
@@ -24,11 +25,6 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
-// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
```
Comment (justinfagnani) [inline src/build/build.ts]: was there a type declaration from npm? If so, remove it. If not, then that's not what was causing the error, it was the lack of a declaration file.

**Comment by justinfagnani** [comment]
LGTM

File: src/build/build.ts (hunk 2 above)
Comment (FredKSchott) [inline src/build/build.ts]: Yea I tried to clarify this in the comment (/w "consider writing a custom declaration in the `custom_typings/` folder"). Could have been more clear.

**Pull request merged**